src/library-authoring/library-access/UserAccessWidget.jsx

```diff
@@ -40,99 +41,121 @@ export const UserAccessWidget = ({
-                <Modal
-                  open={showDeprivModal}
-                  title={intl.formatMessage(messages['library.access.modal.remove_admin.title'])}
-                  onClose={() => setShowDeprivModal(false)}
-                  body={(
-                    <div>
-                      <p>
-                        {intl.formatMessage(
-                          messages['library.access.modal.remove_admin.body'],
-                          { library: library.title, email: user.email },
-                        )}
-                      </p>
-                    </div>
-                  )}
-                  buttons={[
-                    <Button
-                      onClick={() => setAccessLevel(LIBRARY_ACCESS.AUTHOR).then(setShowDeprivModal(false))}
+            <Col xs={12} md={6}>
+              <Row>
+                {(user.access_level === LIBRARY_ACCESS.ADMIN) && adminLocked && (
+                  <Col xs={12} className="text-center text-md-right">
+                    <small>{intl.formatMessage(messages['library.access.info.admin_unlock'])}</small>
+                  </Col>
+                )}
+                {user.access_level === LIBRARY_ACCESS.ADMIN && multipleAdmins && (
+                  <Col xs={10} className="text-left text-md-right">
+                    <Button size="lg" variant="secondary" onClick={() => setShowDeprivModal(true)}>
+                      {intl.formatMessage(messages['library.access.remove_admin'])}
+                    </Button>
+                    <ModalDialog
+                      isOpen={showDeprivModal}
+                      onClose={() => setShowDeprivModal(false)}
                     >
-                      {intl.formatMessage(commonMessages['library.common.forms.button.yes'])}
-                    </Button>,
-                  ]}
-                />
-              </Col>
-              )}
-              {user.access_level === LIBRARY_ACCESS.READ && (
-              <Col xs={10} className="text-left text-md-right">
-                <Button size="lg" variant="primary" onClick={() => setAccessLevel(LIBRARY_ACCESS.AUTHOR)}>
-                  {intl.formatMessage(messages['library.access.add_author'])}
-                </Button>
-              </Col>
-              )}
-              {user.access_level === LIBRARY_ACCESS.AUTHOR && (
-                <>
-                  <Col xs={5} className="text-left text-md-right pl-md-1">
-                    <Button size="lg" variant="secondary" onClick={() => setAccessLevel(LIBRARY_ACCESS.READ)}>
-                      {intl.formatMessage(messages['library.access.remove_author'])}
+
+                      <ModalDialog.Header>
+                        <ModalDialog.Title>
+                          {intl.formatMessage(messages['library.access.modal.remove_admin.title'])}
+                        </ModalDialog.Title>
+                      </ModalDialog.Header>
+                      <ModalDialog.Body>
+                        <div>
```

Comment:
same as previous comment about probably not needing this `<div>`

Reply:
Resolved